src/Core/Addon/Theme/ThemeManager.php

```diff
@@ -372,4 +383,113 @@ public function saveTheme($theme)
 
         file_put_contents($jsonConfigFolder.'/shop'.$this->shop->id.'.json', json_encode($theme->get(null)));
     }
+
+    /**
+     * Import translation from Theme to Database
+     *
+     * @param $theme
```

Comment:
Please don't forget to specify type and description for all parameters when possible.